tool/src/org/antlr/v4/Tool.java

```diff
@@ -117,24 +119,24 @@ public Option(String fieldName, String name, OptionArgType argType, String descr
 	public boolean longMessages = false;
 
     public static Option[] optionDefs = {
-        new Option("outputDirectory",	"-o", OptionArgType.STRING, "specify output directory where all output is generated"),
```

Comment:
Is this file just whitespace changes? It's hard to see

Reply:
Yes, while I was on this I also fixed formatting. This part was pretty much broken.